packages/composites/market-closure/schemas/env.json

```
@@ -0,0 +1,26 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/market-closure-adapter.json",
+  "title": "@chainlink/market-closure-adapter env var schema",
+  "type": "object",
+  "properties": {
+    "FCSAPI_DATA_PROVIDER_URL": {
+      "type": "string",
+      "format": "uri"
+    },
+    "FINNHUB_DATA_PROVIDER_URL": {
+      "type": "string",
+      "format": "uri"
+    },
+    "CHECK_API_KEY": {
+      "type": "string"
+    },
+    "TH_API_KEY": {
+      "type": "string"
+    }
+  },
```

```suggestion
  },
  "required": ["RPC_URL"],
```
Also needs "one of" the DATA_PROVIDER_URLs, not sure if we can do that here.